lib/grant-types/refresh-token-grant-type.js

```diff
@@ -130,7 +130,7 @@ RefreshTokenGrantType.prototype.revokeToken = function(token) {
   return promisify(this.model.revokeToken, 1).call(this.model, token)
     .then(function(status) {
       if (!status) {
-        throw new InvalidGrantError('Invalid grant: refresh token is invalid');
+        throw new InvalidGrantError('Invalid grant: refresh token is invalid, no status');
```

Comment:
`status` seems to be the result of the call to revokeToken-method of the model. So writing "no status" is confusing. If status is true, we skip throwing the error, if status is false, we throw. So this is handling the case, that when the model returns false, that the refresh token was not right or we could for some reason not revoke the token. 

```suggestion
        throw new InvalidGrantError('Invalid grant: refresh token is invalid or could not be revoked');
```

Reply:
I have absolutely no preference on the verbiage of the error only that its different. Mostly because it took me entirely toooooo long yesterday to find an error in one of my projects.